sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1495,6 +1495,53 @@ pi_result piDevicesGet(pi_platform Platform, pi_device_type DeviceType,
   return PI_SUCCESS;
 }
 
+// sub-sub-device
+// TODO: do we need to gather Ordinals or just the SubSubDevicesCount will do ?
+pi_result getCmdQueueOrdinals(pi_device PiSubDevice, pi_uint32 &SubSubDevicesCount, std::vector<int>& Ordinals,
+                              std::vector<ze_command_queue_group_properties_t>& AllQueueProperties) {
+  uint32_t numQueueGroups = 0;
+  ZE_CALL(zeDeviceGetCommandQueueGroupProperties,
+          (PiSubDevice->ZeDevice, &numQueueGroups, nullptr));
+  if (numQueueGroups == 0) {
+    return PI_ERROR_UNKNOWN;
+  }
+  std::vector<ze_command_queue_group_properties_t> QueueProperties(
+      numQueueGroups);
+  ZE_CALL(zeDeviceGetCommandQueueGroupProperties,
+          (PiSubDevice->ZeDevice, &numQueueGroups, QueueProperties.data()));
+
+  SubSubDevicesCount = numQueueGroups;
```

Comment:
this will be the total # of groups, not only compute ones. not what you want, right?

Reply:
right. will just collect Ordinals and use Ordinals.size() for the count.

Reply:
done.